src/sonic-device-data/tests/media_checker

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
```

Comment:
this is not automatically run? can you embed into the build process to make sure it automatically run in the build PR.

Reply:
@lguohan I will check on this.

Reply:
@lguohan Addressed this review comment.